lib/base/logger.hpp

```diff
@@ -22,7 +23,8 @@ enum LogSeverity
 	LogNotice,
 	LogInformation,
 	LogWarning,
-	LogCritical
+	LogCritical,
+	LogNothing
 };
```

Comment:
Is the latter such a big problem? (The only thing where I've ever always put commas is Puppet.)

Reply:
Define "such a big problem". If that comma already where there, the diff would just show `+LogNothing,` now instead of three lines. So I see an advantage in doing it that way.